nano/scripts/nano/nano_state_pda.js

```diff
@@ -0,0 +1,57 @@
+NanoStatePDAClass.inheritsFrom(NanoStateClass);
+var NanoStatePDA = new NanoStatePDAClass();
+var pda_app_template = null;
+
+function NanoStatePDAClass() {
+    this.key = 'pda';
+    this.key = this.key.toLowerCase();
+
+    NanoStateManager.addState(this);
+}
+	
+NanoStatePDAClass.prototype.onUpdate = function(data) {
+	NanoStateClass.prototype.onUpdate.call(this, data);
+	
+	try {
+		if(data['data']['app'] != null) {
+			var template = data['data']['app']['template'];
+			if(template != null && template != pda_app_template) {
+				$.when($.ajax({
+					url: template + '.tmpl',
+					cache: false,
+					dataType: 'text'
+				}))
+				.done(function(templateMarkup) {
+					templateMarkup += '<div class="clearBoth"></div>';
+
+					try {
+						NanoTemplate.addTemplate('app', templateMarkup);
```

Comment:
`resetTemplate` clears the cached, parsed versions of the templates. If you don't call that, it'll use the first version that was loaded, whereas I'm changing the template here of course.